### ISSUE TITLE
vue-translate crashes on IE11 when trying to translate text passing in as prop

### ISSUE THREAD

**Issue by Jarzka** [javisperez/vuetranslate#6]
Hi

I have the following line on my Vue component:

> <span class="omo-purchase-frequency-header__info__time" v-translate>{{ infoText }}</span>

While this component is rendered for the first time on IE11, it works. However, when the component is rendered again with different property, IE crashes with the following errors:

> [Vue warn] Error in nextTick: "Error: Invalid argument:"
> Error: Invalid argument.

This bug does NOT happen if v-translate is removed or the `{{ infoText }} `is replaced with a hardcoded text. In other words, this bug only happens when v-translate is used to translate a text passed in as a property. I also checked that infoText is never null, all the possible values that are used as intoText and text values and there is a translation for all of them. Also, this bug does not happen on Firefox and Chrome, only IE11.

**Comment by Jarzka** [comment]
Here is a shot of the IE error:
https://imgur.com/a/BckqfSZ

**Comment by Jarzka** [comment]
It seems that this problem can be avoided by writing  ````<span v-translate class="..." v-text="infoText">blaablaa</span>```` instead of using `{{ infoText }}`

**Comment by Jarzka** [comment]
Actually, I was wrong. Using v-text never crashes but the prop is also never translated. I went back to  using `{{ infoText}}`  and noticed that even though Chrome and Firefox do not crash, the property is never translated.

**Comment by javisperez** [comment]
Wow 🤔 this is a funky one, i need to debug this deeper, i don't really have access to IE11 but i'll try to get a VM and make some tests, thanks for reporting and for your patience 🙏

**Comment by Jarzka** [comment]
Yes, it is. Anyway, no hurry to fix it anymore. Our app hit production 11 months ago using i18 library. We would have sticked with v-translate, but unfortunately this bug forced us to change library.
